com.github.philip_scott.spice-up.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "com.github.philip_scott.spice-up",
-    "base": "io.elementary.Loki.BaseApp",
+    "base": "io.elementary.Juno.BaseApp",
 	"base-version": "stable",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "1.6",
```

Comment:
This must be updated to `18.08`. The 1.6 release is "outdated" see https://blogs.gnome.org/alexl/2018/10/11/moving-away-from-the-1-6-freedesktop-runtime/